coalahtml/helper.py

```diff
@@ -0,0 +1,108 @@
+import os
+import shutil
+import socket
+
+from coalib.parsing.DefaultArgParser import default_arg_parser
+from coalahtml.tree import Tree
+
+COALA_HTML_BASE = '_coalahtml'
+COALA_HTML_DATA = 'data'
+CONFIGS = {
+        "results_file": os.path.join(COALA_HTML_DATA, 'coala.json'),
+        "file_data": os.path.join(COALA_HTML_DATA, 'file_data.json'),
+        "files": os.path.join(COALA_HTML_DATA, 'files.json'),
+        "home": os.path.join(COALA_HTML_DATA, 'roothome')}
```

Comment:
Maybe it'll be good idea to save config in one file both for js and python code?